srcpkgs/julia/template

```diff
@@ -1,36 +1,36 @@
 # Template file for 'julia'
 pkgname=julia
-version=1.6.1
+version=1.7.1
 revision=1
 archs="i686* x86_64* armv7l* aarch64*"
 build_style=gnu-makefile
 make_build_args="prefix=/usr sysconfdir=/etc datarootdir=/usr/share
  USE_BINARYBUILDER=0 USE_SYSTEM_LIBM=0 USE_SYSTEM_DSFMT=0 USE_SYSTEM_LIBUV=0
- USE_SYSTEM_SUITESPARSE=0 USE_SYSTEM_BLAS=0 USE_SYSTEM_LAPACK=0 USE_SYSTEM_LLVM=0
+ USE_SYSTEM_LIBSUITESPARSE=0 USE_SYSTEM_BLAS=0 USE_SYSTEM_LAPACK=0 USE_SYSTEM_LLVM=0
  USE_SYSTEM_LIBUNWIND=1 USE_SYSTEM_PCRE=1 USE_SYSTEM_GMP=1 USE_SYSTEM_PATCHELF=1
  USE_SYSTEM_MPFR=1 USE_SYSTEM_MBEDTLS=1 USE_SYSTEM_LIBSSH2=1 USE_SYSTEM_NGHTTP2=1
  USE_SYSTEM_CURL=1 USE_SYSTEM_ZLIB=1 USE_SYSTEM_P7ZIP=1 USE_SYSTEM_LIBGIT2=1
- USE_SYSTEM_UTF8PROC=1
- UTF8PROC_INC='${XBPS_CROSS_BASE}/usr/include'"
+ USE_SYSTEM_UTF8PROC=1 USE_SYSTEM_LIBBLASTRAMPOLINE=1 USE_SYSTEM_CSL=1 USE_SYSTEM_LIBWHICH=1"
 make_install_args="$make_build_args"
 make_check_args="$make_build_args"
 make_check_target=testall
 conf_files="/etc/julia/startup.jl"
-hostmakedepends="pkg-config perl cmake python gcc-fortran patchelf which tar xz"
+hostmakedepends="pkg-config perl cmake python3 gcc-fortran patchelf which tar xz libwhich"
```

Comment:
I wonder if `libwhich` here is complicating the search for libraries in a way that triggers vendored builds of libraries you've installed in `makedepends`.